win/CS/HandBrakeWPF/Services/Encode/Factories/EncodeTaskFactory.cs

```diff
@@ -327,6 +327,10 @@ private static Video CreateVideo(EncodeTask job, HBConfiguration configuration)
                 {
                     video.Options = string.IsNullOrEmpty(video.Options) ? "lowpower=1" : ":lowpower=1";
                 }
+                else if(!configuration.EnableQsvLowPower && !video.Options.Contains("lowpower"))
+                {
+                    video.Options = string.IsNullOrEmpty(video.Options) ? "lowpower=0" : ":lowpower=0";
```

Comment:
Why do we need to explicitly set lowpower=0?

Reply:
If checkbox is "unchecked", we need to change default behavior for newer cpus

Reply:
You seem to have set " param->videoParam->mfx.LowPower     = MFX_CODINGOPTION_OFF; "
above. Does this not have the same effect?

Reply:
Yes, it has the same effect.